app/views/admin/dashboard/_admin_dashboard.html.slim

```diff
@@ -0,0 +1,28 @@
+- provide(:title, 'Admin dashboard')
+.row
+  .col-lg-12
+    .page-header
+      h2.center Admin dashboard
+.row
+  = render 'admin/dashboard/panel', count: Question.count,

```

Comment:
:thumbsup: for partials
just few notes:
- better avoid using local names like "count" - they can be confusing
- use boolean instead of "yes/no" :)